Gemfile

```diff
@@ -5,25 +5,25 @@ gem "sinatra", "1.3.4"
 gem "rake", "~> 10.5"
 gem "rack", "~> 1.6"
 gem "rest-client", "1.8.0"
-gem "logging", "1.8.1"
-gem 'nokogiri', "1.5.5"
+gem "logging", "2.1.0"
+gem 'nokogiri', "1.6.7.2"
 gem 'whenever', require: false
-gem "slop", "3.4.5"
-gem "sidekiq", "2.13.0"
+gem "slop", "4.3.0"
+gem "sidekiq", "4.1.1"
 gem "sidekiq-statsd", "0.1.5"
 # pin to version that includes security vulnerability fix
```

Comment:
It refers to `redis-namespace`, which is now updated so it's definitely not needed anymore.